protocol/river-control-unstable-v1.xml

```diff
@@ -63,7 +63,7 @@
       by the compositor after one of the events is sent.
     </description>
 
-    <event name="success">
+    <event name="success" type="destructor">
```

Comment:
This change is fine to make without a version bump, the only reason this annotation wasn't here from the start is that it wasn't supported for events yet when I wrote this protocol: https://gitlab.freedesktop.org/wayland/wayland/-/merge_requests/82.